prog/vm/github_runner.rb

```diff
@@ -245,6 +245,31 @@ def setup_info
       COMMAND
     end
 
+    if github_runner.installation.project.get_ff_runner_jit_file
+      # After testing in production for a while, we can move these files to the image generation process
+      command += <<~COMMAND
+        sudo tee /etc/systemd/system/runner-script.service > /dev/null > /dev/null <<'EOT'
```

Comment:
```suggestion
        sudo tee /etc/systemd/system/runner-script.service > /dev/null 2>&1 <<'EOT'
```

Assuming you are trying to redirect both stdout and stderr to /dev/null.

Reply:
If something goes wrong, I think we should see the `stderr`.

Reply:
I only redirect stdout because tee also prints the file content.